docs/api-content/api-docs/1-introduction.md

```diff
@@ -8,8 +8,8 @@ sidebar_custom_props:
   icon: "graph"
 ---
 
-The API documentation section includes documentation for Palette API and Edge Host API. Edge Host API endpoints are Tech
-Preview features and should not be used in production workloads.
+The API documentation section includes documentation for Palette API and Local Management API. Local Management API
```

Comment:
@lennessyy could we remove the tech preview language and instead apply the Tech Preview admonition in the Local Management API section? That way when we remove the tech preview for the feature, it will be easier to make the update?